templates/partials/payroll_expenditure.html

```diff
@@ -1,5 +1,7 @@
 <div class="card-body">
-  <h4 class="card-subtitle">Payroll Expenditure Breakout</h4>
+  <h4 class="card-subtitle">Payroll expenditure breakout</h4>
+  <h4 class="card-subtitle"><small class="text-muted">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
```

Comment:
How about adding a little daylight above and below the subtitle? You can use [Bootstrap's handy dandy spacing utilities](https://getbootstrap.com/docs/4.0/utilities/spacing/).